src/easypost.js

```diff
@@ -221,16 +232,47 @@ export default class EasyPostClient {
       request.auth(this.key);
     }
 
-    if (params !== {} && params !== undefined) {
```

Comment:
Got rid of the `params !== {}` check because it is always true

Reply:
How so? It defaults to `{}` and there are certainly HTTP call functions (e.g. retrieve) that don't have any query/body data and therefore no `params`.